tests/models/wav2vec2_bert/test_processor_wav2vec2_bert.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2021 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import shutil
+import tempfile
+import unittest
+
+from transformers.models.seamless_m4t import SeamlessM4TFeatureExtractor
+from transformers.models.wav2vec2 import Wav2Vec2CTCTokenizer
+from transformers.models.wav2vec2.tokenization_wav2vec2 import VOCAB_FILES_NAMES
+from transformers.models.wav2vec2_bert import Wav2Vec2BERTProcessor
+from transformers.utils import FEATURE_EXTRACTOR_NAME
+
+from ..wav2vec2.test_feature_extraction_wav2vec2 import floats_list
+
+
+class Wav2Vec2BERTProcessorTest(unittest.TestCase):
```

Comment:
Any test methods here which are the same for wav2vec2  processor should have copied from statements here